dragonchain/broadcast_processor/broadcast_processor.py

```diff
@@ -112,6 +132,8 @@ def make_broadcast_futures(session: aiohttp.ClientSession, block_id: str, level:
             headers, data = authorization.generate_authenticated_request("POST", chain, path, broadcast_dto)
             if level != 5:
                 headers["deadline"] = str(BROADCAST_RECEIPT_WAIT_TIME)
+            else:
+                headers["deadline"] = str(get_l5_wait_time(chain))
```

Comment:
this means that for every l5 broadcast, you are now retrieving from matchmaking and calculating the l5 wait time twice; once in this function, and once in the main broadcast loop for re-scheduling its check. You should only have this in one spot, especially considering it's making an external call to calculate

Reply:
Fixed, as discussed.